src/profile-logic/call-tree.js

```diff
@@ -156,7 +156,7 @@ export class CallTree {
   }
 
   hasChildren(callNodeIndex: IndexIntoCallNodeTable): boolean {
-    return this.getChildren(callNodeIndex).length !== 0;
+    return this._callNodeHasChildren[callNodeIndex] !== 0;
```

Comment:
I guess we could have used the previous childCount too, but I'm not opposed to changing childCount to hasChildren